blackbelt/commands/dep.py

```diff
@@ -13,9 +13,12 @@ def cli():
 
 
 def validate_dep(ctx, param, dep):
+    if os.path.isdir(dep): # check the provided project_dir
+        click.echo('The dependency is a local directory, using whatever is currently in node_modules')
+        if not os.path.isdir(os.path.join(dep, 'node_modules')):
+            raise click.BadParameter('There are no node_modules to analyze in the `'+dep+'` project directory')
+        return (dep, '*')
```

Comment:
If I understand it correctly, the `*` is just a placeholder here to let the other part of the code know the first item in the tuple is a relative path? In that case I'd use `(dep, None)`, which doesn't look like a magical npm-specific syntax with some special meaning.

Beware, if you use that, then the comparison needs to be a bit different:

```python
if dep_version is None:
```